martin-mbtiles/src/tile_copier.rs

```diff
@@ -241,6 +280,14 @@ impl TileCopier {
 
         Ok(())
     }
+
+    async fn detect_type(&self, mbtiles: &Mbtiles) -> MbtResult<MbtType> {
```

Comment:
i moved this function out of the `impl` block because it doesn't need `self` (not certain why clippy hasn't complained)